### DISCUSSION TITLE
Role of json_util.py utility in the repo

### DISCUSSION THREAD

**Discussion by Solomon1732** [Tribler/tribler#6012] [General] [answered]
As replied by @devos50 [here](https://github.com/Tribler/anydex-core/issues/62#issuecomment-781519982), the json_util.py file in anydex-core came to be when the repo was migrated from Py2 to Py3. Is it the same case in repo?

Edit: opened a discussion page after being redirected here from the Tribler forum https://forum.tribler.org/t/role-of-json-util-py-utility-in-tribler-repo/6478

**Comment by qstokkink** [comment, marked as answer]
When JSON decoding failed in Python 2 we would be confronted with a ["something went wrong 🤷" error](https://github.com/Tribler/tribler/issues/3059). That is why I made `json_util.py` 4 years ago.

Now that we are running on Python 3, the file is no longer necessary. JSON decoding errors are now much richer in context. Downstream, in IPv8, the `json_util.py` has also long since been removed.

**Reply by Solomon1732** [reply]
Is there a chance for it to be removed from this repo as well?

**Reply by qstokkink** [reply]
Certainly, this file could've been removed a year ago already. I simply forgot about it 😃

**Reply by Solomon1732** [reply]
Should I open an issue about it?

**Reply by devos50** [reply]
Feel free to open an issue 👍
